src/component/molecules/menuWrapper/menuWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import { Box } from "@mui/material";
+import { FC, useContext } from "react";
+import PopupsMenu from "../../organism/popupsMenu/popupsMenu";
+import { NavbarMenuContext } from "../../../context/navbarMenuContext";
+
+const MenuWrapper: FC = () => {
+    const {categories} = useContext(NavbarMenuContext)
+    return (
+        <Box>
+            <PopupsMenu objArr={categories}/>
```

Comment:
`objArr` is not a good name for this property. Change that on something that better describes.